snyk/cli
# chore: Remove new output feature flag [CFG-2019]

#### What does this PR do?

- Removes the `iacCliOutputRelease` feature flag.
- Removes the legacy output paths, since they're no longer reachable.

#### How should this be manually tested?
- Run `snyk iac test` and ensure the test results are outputted as expected in all formats.

#### Any background context you want to provide?

- Once the New CLI Output has been in GA for 1 month, we would like to remove the `iacCliOutputRelease` feature flag from the code altogether.

#### What are the relevant tickets?

- [CFG-2019](https://snyksec.atlassian.net/browse/CFG-2019)
